### DISCUSSION TITLE
how do i get my microsoft pen to work with fedora?

### DISCUSSION THREAD

**Discussion by IsAmogusDead** [linux-surface/linux-surface#602] [Q&A]
i paid a fuck-ton of money for that pen, and literally never used it.
please help me

**Comment by Vishram1123** [comment]
Have you tried enabling IPTSD after installing the Linux-Surface kernel?

**Reply by IsAmogusDead** [reply]
i did. didn't really seem to do anything, dont know if i'm supposed to do anything else. thank's anyway's
edit: surface pen is not supported :(
